### PR TITLE
Export clerk-sdk-node types from nextjs/api

### PR DESCRIPTION
**Pull request by nikosdouvlis**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/backend-core`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/edge`
- [ ] `@clerk/shared`
- [ ] `build/tooling/chore`

## Description

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.

### CODE REVIEW

**Review by anagstef — APPROVED** [review]
Looks good!

**Comment by SokratisVidros** [comment]
@nikosdouvlis Let's add a reminder in ClerkJS V5 release to deprecate the Node SDK exports

**Comment by nikosdouvlis** [comment]
> @nikosdouvlis Let's add a reminder in ClerkJS V5 release to deprecate the Node SDK exports

Already working on writing a doc with the final v5 api

File: packages/nextjs/src/api/index.ts [hunk 1]
```diff
@@ -35,4 +43,6 @@ export {
   organizations,
   clients,
   allowlistIdentifiers,
+  createClerkClient,
+  clerkClient,
```
Comment (dimkl) [inline packages/nextjs/src/api/index.ts:47]: ❓ Based on the title i was expecting the addition of some types in the export. I see that we have also exported the  `clerkClient`. Is it intentional?

**Pull request merged**